option-parser.js

```diff
@@ -16,27 +17,74 @@ const defaultTargets = [
 const pkg = pkgRoot();
 let enact = pkg.meta.enact || {};
 
-function gentlyParse(file) {
+const capitalize = name => name.charAt(0).toUpperCase() + name.slice(1);
+const valid = v => v || typeof v === 'boolean';
```

Comment:
I worry that this valid function may be re-used in places where an empty string or `0` could be 'valid'.  If it's the case that we don't want to allow those, maybe a comment or, perhaps, the (only slightly) more explicit `v || v === false`?